lib/cli/bin/generate.js

```diff
@@ -66,102 +70,164 @@ const end = () => {
+        .then(commandLog('Adding storybook support to your "Create React App" based project'))
+        .then(end);
+      break;
+
+    case types.REACT:
+      // eslint-disable-next-line
+      require('../generators/REACT')
+        .then(commandLog('Adding storybook support to your "React" app'))
+        .then(end);
+      break;
+
+    case types.REACT_NATIVE_SCRIPTS: {
+      const app = chalk.bold('"./App.js"');
+      // eslint-disable-next-line
+      require('../generators/REACT_NATIVE_SCRIPTS')
+        .then(commandLog('Adding storybook support to your "Create React Native App" app'))
+        .then(end)
+        .then(() => {
+          logger.log(chalk.red('NOTE: CRNA app installation is not 100% automated.'));
+          logger.log(`To quickly run storybook, replace contents of ${app} with:\n`);
+          codeLog(["export default from './storybook';"]); // eslint-disable-line
+          logger.log('\nFor a more complete discussion of options, see:\n');
+          logger.log(chalk.cyan(CRNA_DISCUSSION));
+          logger.log();
+        });
+      break;
+    }
+
+    case types.REACT_NATIVE:
+      // eslint-disable-next-line
+      require('../generators/REACT_NATIVE')
+        .then(commandLog('Adding storybook support to your "React Native" app'))
+        .then(end);
+      break;
+
+    case types.METEOR:
+      // eslint-disable-next-line
+      require('../generators/METEOR')
+        .then(commandLog('Adding storybook support to your "Meteor" app'))
+        .then(end);
+      break;
+
+    case types.WEBPACK_REACT:
+      // eslint-disable-next-line
+      require('../generators/WEBPACK_REACT')
+        .then(commandLog('Adding storybook support to your "Webpack React" app'))
+        .then(end);
+      break;
+
+    case types.REACT_PROJECT:
+      // eslint-disable-next-line
+      require('../generators/REACT')
+        .then(commandLog('Adding storybook support to your "React" library'))
+        .then(end);
+      break;
+
+    case types.SFC_VUE:
+      // eslint-disable-next-line
+      require('../generators/SFC_VUE')
+        .then(commandLog('Adding storybook support to your "Single File Components Vue" app'))
+        .then(end);
+      break;
+
+    case types.VUE:
+      // eslint-disable-next-line
+      require('../generators/VUE')
+        .then(commandLog('Adding storybook support to your "Vue" app'))
+        .then(end);
+      break;
+
+    default:
+      paddedLog(`We couldn't detect your project type. (code: ${projectType})`);
+      paddedLog('Check that you are at your project directory.');
+      paddedLog(
+        'You can bootstrap a project using some of the options below, or follow one of the slow start guides: https://storybook.js.org/basics/slow-start-guide/'
+      );
+      inquirer
+        .prompt([
+          {
+            type: 'list',
+            name: 'type',
+            message: 'Which kind of project would you like to bootstrap?',
```

Comment:
`Which kind of project boilerplate would you like to install and start?`

I think `install` is pretty key here since we are going installing something into the user's system without their acknowledgement here.